PyPDF2/_page.py

```diff
@@ -1106,6 +1106,7 @@ def _extract_text(
         self,
         obj: Any,
         pdf: Any,
+        orientations: Tuple[int] = (0, 90, 270, 360),
```

Comment:
That default does not match the type signature. For that default, the correct type would be: `Tuple[int, int, int, int]`. If you want a tuple with arbitray arguments, you need `Tuple[int, ...]` (yes, really with the dots :-) )

Alternatively, I guess you want to allow only those 4 numbers. Then `Literal[0, 90, 270, 360]` would be the correct type.

Reply:
By the way, that can also be combined, e.g.:

`Tuple[Literal[0, 90, 270, 360], ...]`

would mean you have a tuple of arbitrary many integers, but every integer has to be one of 0 / 90 / 270 / 360

Reply:
not exactly : What I'm proposing is a variable length tuple or an int):
page.extract_text(0) => extract all text strings oriented up
page.extract_text((0,)) => extract all text strings oriented up (synonym)
page.extract_text((0,180)) => extract all text strings oriented up or down